src/awkward/_broadcasting.py

```diff
@@ -841,7 +841,8 @@ def continuation():
                         else:
                             lencontent = backend.index_nplike.max(stops)
                             nextinputs.append(x.content[:lencontent])
-
+                    elif isinstance(x, RegularArray):
+                        nextinputs.append(x.content[: x.size * x.length])
```

Comment:
We test for `RegularArray` in `all_same_offsets`, but here we weren't actually pulling out the content.